components/emoji_picker/emoji_picker.test.js

```diff
@@ -171,6 +157,58 @@ describe('DtEmojiPicker Tests', () => {
       expect(fixedLabel.text()).toEqual(tabSetLabels[0]);
     });
 
+    describe('Skin tone selector tests', () => {
+      it('Should render provided skin selector button tooltip label', () => {
+        const skinToneSelector = wrapper.find('.d-emoji-picker__skin-selected button');
+
+        expect(skinToneSelector.exists()).toBe(true);
+        expect(skinToneSelector.attributes('aria-label')).toBe(skinSelectorButtonTooltipLabel);
+      });
+
+      it('Should render provided skin tone in skin tone selector', () => {
+        const skinToneSelector = wrapper.find('.d-emoji-picker__skin-selected button img');
+
+        expect(skinToneSelector.exists()).toBe(true);
+        expect(skinToneSelector.attributes('aria-label')).toBe(':wave_tone1:');
+      });
+
+      it('Should display skin tone selected from selector', async () => {
+        const skinToneSelectorButton = wrapper.find('.d-emoji-picker__skin-selected button');
+        const skinToneSelectorImg = wrapper.find('.d-emoji-picker__skin-selected button img');
```

Comment:
Create and move all wrapper.find inside

```
const _setChildWrappers = () => {
     const skinToneSelectorButton = wrapper.find('.d-emoji-picker__skin-selected button');
     const skinToneSelectorImg = wrapper.find('.d-emoji-picker__skin-selected button img');
     const skinToneMediumLightButton = wrapper.find('.d-emoji-picker__skin-list').findAll('button').at(1);
     const skinToneSelectorButton = wrapper.find('.d-emoji-picker__skin-selected button');
     const skinToneSelectorImg = wrapper.find('.d-emoji-picker__skin-selected button img');
     const skinToneMediumLightButton = wrapper.find('.d-emoji-picker__skin-list').findAll('button').at(1);
  };
```

then call it after inside the setWrapper function